README.md

```diff
@@ -1 +1,134 @@
-# Car-expense-tracker
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Car-expense-tracker <a name="about-project"></a>
+
+**Car-expense-tracker** keeps track of all your expenses related to your car. It will allow you to keep track of road taxes, insurance and maintenance costs.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- CSS
+- HTML
+- Ruby 
+- Ruby on Rails
+- Postgresql
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **calculate fuel mpg**
+- **set maintenance dates**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+This application currently has no live demo.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need Ruby and  Postgresql installed on your computer.
+
+### Setup
+
+Clone this repository to your desired folder: 
+
+`git clone https://github.com/vflorin1995/Car-exoense-tracker.git`
+
+
+### Install
+
+Then install the project's necessary dependencies with: 
+
+`bundle install`
+
+
```

Comment:
**[Optional]**

- [ ] In the Getting Started Section you should have the following commands in their respective sub-sections to make your readme file look professional and descriptive :+1:

### Setup

```
git clone https://github.com/vflorin1995/Car-exoense-tracker.git
```
### Install

```
bundle install
```
### Setup Database
```
rails db:create
rails db:migrate
```
### Usage

```
rails server
```

### Test

```
rspec spec
```